variant-remapping-automation/remapping_automation.py

```diff
@@ -61,28 +62,58 @@ def write_remapping_process_props_template(template_file_path):
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 logging.level.uk.ac.ebi.eva.accession.remapping=INFO
 parameters.chunkSize=1000
+''')
+        return template_file_path
+
+    @staticmethod
+    def write_clustering_props_template(template_file_path, instance):
+        # Additional properties needed for clustering; these need to be appended to the above.
+        properties = get_properties_from_xml_file(cfg['maven']['environment'], cfg['maven']['settings_file'])
+        counts_url = properties['eva.count-stats.url']
+        counts_username = properties['eva.count-stats.username']
+        counts_password = properties['eva.count-stats.password']
+        with open(template_file_path, 'w') as open_file:
+            open_file.write(f'''
+accessioning.instanceId=instance-{instance}
+accessioning.submitted.categoryId=ss
+accessioning.clustered.categoryId=rs
+
+accessioning.monotonic.ss.blockSize=100000
+accessioning.monotonic.ss.blockStartValue=5000000000
+accessioning.monotonic.ss.nextBlockInterval=1000000000
+accessioning.monotonic.rs.blockSize=100000
+accessioning.monotonic.rs.blockStartValue=3000000000
+accessioning.monotonic.rs.nextBlockInterval=1000000000
+
+eva.count-stats.url={counts_url}
+eva.count-stats.username={counts_username}
+eva.count-stats.password={counts_password}
 ''')
         return template_file_path
 
     def get_job_information(self, assembly, taxid):
         query = (
             'SELECT source, scientific_name, assembly_accession, remapping_status, SUM(num_studies), '
```

Comment:
I'm wondering if we should keep the nb of studies if we mandate that study accessions should be provided. 
Although in some case we will likely prefer not giving the list of study to process them all and then the number of studies will be useful.

Reply:
That's what I was thinking, it's a bit redundant but maybe still useful in the latter case...